node-rfc/cts

```diff
@@ -1,13 +1,12 @@
 #! /bin/sh
 
-
 $SAPBIN/node_modules/grunt-cli/bin/grunt --stack --no-color --gruntfile $SAPBIN/run_rfc_task.js "$@" 1>&2
 RC=$?
 
 # In case a transport request has been created this file provides the ID of
 # the transport request in a json format like {"REQUESTID":"ZZZK9NNNNN"}
-TRANSPORT_REQUEST_FILE="${SAPDATADIR}/CTS_Data.txt"
-TRACE_FILE=dev_rfc.trc
+TRANSPORT_REQUEST_FILE="${HOME}/CTS_Data.txt"
+TRACE_FILE="${HOME}/dev_rfc.trc"
```

Comment:
I am not sure if that is correct. The trace file appeared in the past in the working directory.